### ISSUE TITLE
Lottie exporting with missing lines

### ISSUE THREAD

**Issue by pamelagray** [airbnb/lottie-web#2511]
Hello, hoping someone can help me.
I have been creating short optical lottie animations using simple shape layers but have also been using a plug-in called ‘Connect Layers Pro’ which attaches shape layer lines (‘chains’) to other shapes. I wasn’t sure if this would work with lottie and tested it and it exported perfectly but now my exports are missing these lines whenever I use the same technique, has something changed?

**Comment by bodymovin** [comment]
hi, what version of bodymovin and lottie are you using?

**Comment by pamelagray** [comment]
Think it’s the latest update

**Comment by bodymovin** [comment]
can you share an .aep project so I can test it?

**Comment by pamelagray** [comment]
Hi, sure here is a link: https://we.tl/t-7S7KZZdGHv
I'm thinking it's probably the plug-in causing issues but unsure why it worked before.

**Comment by bodymovin** [comment]
yes, unfortunately it's using a lot of expressions and some  don't work well in this scenario, specially with repeaters.
As a workaround, you can export the animation with expressions converted to keyframes, by selecting the cog next to the composition in bodymovin, and then going to:
Expression options > Convert expressions to keyframes
Don't select to include hidden layers, since you don't need them.
